gemfiles/ruby_3.4_relational_db.gemfile.lock

```diff
@@ -27,22 +27,24 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activemodel (7.1.3.4)
-      activesupport (= 7.1.3.4)
-    activerecord (7.1.3.4)
-      activemodel (= 7.1.3.4)
-      activesupport (= 7.1.3.4)
+    activemodel (7.2.2)
+      activesupport (= 7.2.2)
+    activerecord (7.2.2)
+      activemodel (= 7.2.2)
+      activesupport (= 7.2.2)
```

Comment:
We need ActiveRecord 7.1.4+ to be able to use sqlite 2.0+, per https://github.com/rails/rails/blob/v7.1.3.4/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L14 and https://github.com/rails/rails/blob/v7.1.4/activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb#L14